assembly/src/parsers/ast/serde/mod.rs

```diff
@@ -223,21 +223,23 @@ pub enum OpCode {
     MemStoreWImm = 196,
     LocStoreW = 197,
 
-    AdvPush = 198,
-    AdvLoadW = 199,
+    MemStream = 198,
 
-    AdvU64Div = 200,
+    AdvPush = 199,
+    AdvLoadW = 200,
+
+    AdvU64Div = 201,
 
     // ----- cryptographic operations ---------------------------------------------------------
-    RPHash = 201,
-    RPPerm = 202,
-    MTreeGet = 203,
-    MTreeSet = 204,
-    MTreeCWM = 205,
+    RPHash = 202,
```

Comment:
We might want to defer these to constant values in the future so its easier to update. Example:

```rust
const RPHASH: _ = ADVU64DIV + 1;
...
RPHash = RPHASH,
```

Reply:
Good idea! Though, I hope these won't change much going forward.